src/webgpu/shader/execution/expression/call/builtin/bitcast.spec.ts

```diff
@@ -0,0 +1,334 @@
+  .desc(`bitcast u32 to u32 tests`)
+  .params(u =>
+    u
+      .combine('inputSource', allInputSources)
+      .combine('vectorize', [undefined, 2, 3, 4] as const)
+      .combine('alias', [false, true])
+  )
+  .fn(async t => {
+    const cases = await d.get('u32_to_u32');
+    await run(t, bitcastBuilder('u32', t.params), [TypeU32], TypeU32, t.params, cases);
+  });
+
+g.test('f32_to_f32')
+  .specURL('https://www.w3.org/TR/WGSL/#bitcast-builtin')
+  .desc(`bitcast f32 to f32 tests`)
+  .params(u =>
+    u
+      .combine('inputSource', allInputSources)
+      .combine('vectorize', [undefined, 2, 3, 4] as const)
+      .combine('alias', [false, true])
+  )
+  .unimplemented();
+
+// To i32 from u32, f32
+g.test('u32_to_i32')
+  .specURL('https://www.w3.org/TR/WGSL/#bitcast-builtin')
+  .desc(`bitcast u32 to i32 tests`)
+  .params(u =>
+    u
+      .combine('inputSource', allInputSources)
+      .combine('vectorize', [undefined, 2, 3, 4] as const)
+      .combine('alias', [false, true])
+  )
+  .fn(async t => {
+    const cases = await d.get('u32_to_i32');
+    await run(t, bitcastBuilder('i32', t.params), [TypeU32], TypeI32, t.params, cases);
+  });
+
+g.test('f32_to_i32')
+  .specURL('https://www.w3.org/TR/WGSL/#bitcast-builtin')
+  .desc(`bitcast f32 to i32 tests`)
+  .params(u =>
+    u
+      .combine('inputSource', allInputSources)
+      .combine('vectorize', [undefined, 2, 3, 4] as const)
+      .combine('alias', [false, true])
+  )
+  .unimplemented();
+
+// To u32 from i32, f32
+g.test('i32_to_u32')
+  .specURL('https://www.w3.org/TR/WGSL/#bitcast-builtin')
+  .desc(`bitcast i32 to u32 tests`)
+  .params(u =>
+    u
+      .combine('inputSource', allInputSources)
+      .combine('vectorize', [undefined, 2, 3, 4] as const)
+      .combine('alias', [false, true])
+  )
+  .fn(async t => {
+    const cases = await d.get('i32_to_u32');
+    await run(t, bitcastBuilder('u32', t.params), [TypeI32], TypeU32, t.params, cases);
+  });
+
+g.test('f32_to_u32')
+  .specURL('https://www.w3.org/TR/WGSL/#bitcast-builtin')
+  .desc(`bitcast f32 to i32 tests`)
+  .params(u =>
+    u
+      .combine('inputSource', allInputSources)
+      .combine('vectorize', [undefined, 2, 3, 4] as const)
+      .combine('alias', [false, true])
+  )
+  .unimplemented();
+
+// To f32 from i32, u32
+g.test('i32_to_f32')
+  .specURL('https://www.w3.org/TR/WGSL/#bitcast-builtin')
+  .desc(`bitcast i32 to f32 tests`)
+  .params(u =>
+    u
+      .combine('inputSource', allInputSources)
+      .combine('vectorize', [undefined, 2, 3, 4] as const)
+      .combine('alias', [false, true])
+  )
+  .unimplemented();
+
+g.test('u32_to_f32')
+  .specURL('https://www.w3.org/TR/WGSL/#bitcast-builtin')
+  .desc(`bitcast u32 to f32 tests`)
+  .params(u =>
+    u
+      .combine('inputSource', allInputSources)
+      .combine('vectorize', [undefined, 2, 3, 4] as const)
+      .combine('alias', [false, true])
+  )
+  .unimplemented();
+
+// 16 bit types
```

Comment:
Should there be unimplemented g.test stanzas here?

Reply:
This is a grouping comment.
There are unimplemented stanzas below.